packages/datadog-plugin-mongodb-core/src/index.js

```diff
@@ -1,14 +1,29 @@
 'use strict'
 
 const DatabasePlugin = require('../../dd-trace/src/plugins/database')
+const coalesce = require('koalas')
 
 class MongodbCorePlugin extends DatabasePlugin {
   static get id () { return 'mongodb-core' }
   static get component () { return 'mongodb' }
   // avoid using db.name for peer.service since it includes the collection name
   // should be removed if one day this will be fixed
   static get peerServicePrecursors () { return [] }
+
+  configure (config) {
+    super.configure(config)
+    this.config.mongoHeartbeatDisabled = coalesce(
+      config.mongoHeartbeatDisabled,
```

Comment:
`mongo` is redundant here

Reply:
Also for most other things we'd say `enabled` and set it to true by default, not the other way around.

Reply:
fair enough, fixed